puma.rb

```diff
@@ -6,12 +6,14 @@
   respawn 'yes'
   pid_file (path / 'tmp/pids/puma.pid').abs
 
-  command (path / 'bin/puma').abs
+  puma_path = (path / 'bin/puma').abs
+  socket_path = (path / "tmp/sockets/puma.socket").abs
```

Comment:
Yes, I'm not the biggest fan of fancypath either. Just keeping things consistent for now.